ffs/manager/manager.go

```diff
@@ -76,8 +74,7 @@ func (m *Manager) Create(ctx context.Context) (ffs.APIID, string, error) {
 
 	log.Info("creating instance")
 	iid := ffs.NewAPIID()
-	is := istore.New(iid, namespace.Wrap(m.ds, istoreNamespace))
-	fapi, err := api.New(ctx, iid, is, m.sched, m.wm, defCidConfig)
+	fapi, err := api.New(ctx, namespace.Wrap(m.ds, datastore.NewKey("ffs/manager/api")), iid, m.sched, m.wm, defCidConfig)
```

Comment:
Each `api` is responsible of creating their own `InstanceStore`. The manager just creates/loads `api`s, not also their dependencies.